app/graphql/types/mutation_type.rb

```diff
@@ -63,6 +63,12 @@ class MutationType < Types::BaseObject
     field :create_topic_category, mutation: Mutations::CreateTopicCategory, null: false
     field :delete_topic_category, mutation: Mutations::DeleteTopicCategory, null: false
     field :update_topic_category, mutation: Mutations::UpdateTopicCategory, null: false
+    field :mark_notification, mutation: Mutations::MarkNotification, null: false
+    field :mark_all_notifications, mutation: Mutations::MarkAllNotifications, null: false
+    field :create_topic_subscription, mutation: Mutations::CreateTopicSubscription, null: false
+    field :delete_topic_subscription, mutation: Mutations::DeleteTopicSubscription, null: false
+    field :create_webpush_subscription, mutation: Mutations::CreateWebpushSubscription, null: false
+    field :delete_webpush_subscription, mutation: Mutations::DeleteWebpushSubscription, null: false
```

Comment:
Shouldn't this read `web_push` and `WebPush` instead? It is the `web push protocol`, after all. 🤔

Reply:
I used `webPush` and later switched to `webpush` as the [gem](https://github.com/zaru/webpush) uses it.

Reply:
They probably named it `webpush` probably just to avoid an underscore.

The gem's README heading reads `WebPush`. 😅 

Let's stick to the standard's naming scheme? The values being stored are related to the Web Push Protocol, and not something that is specific to the gem, right?